app/javascript/app/components/sectors-agriculture/drivers-of-emissions/drivers-of-emissions-data.js

```diff
@@ -0,0 +1,15 @@
+export const HISTORICAL_EMISSIONS = 'HISTORICAL_EMISSIONS';
+export const FUTURE_PATHWAYS = 'FUTURE_PATHWAYS';
+
+export const emissionTabs = [
+  {
+    name: 'HISTORICAL EMISSIONS',
+    value: HISTORICAL_EMISSIONS,
+    disabled: false
```

Comment:
hmm, I think you can remove the `disabled` properties now as it should be set to `false` as a default, but double-check 👌

Reply:
You right :+1:  Done